R/compiled.R

```diff
@@ -74,12 +104,17 @@ run_gcov <- function(path, quiet = TRUE,
 
   gcov_inputs <- list.files(path, pattern = rex::rex(".gcno", end), recursive = TRUE, full.names = TRUE)
   run_gcov_one <- function(src) {
+    if(gcov_br) {
+      args = c(gcov_args, src, "-p", "-b", "-c", "-o", dirname(src))
+    } else {
+      args = c(gcov_args, src, "-p", "-o", dirname(src))
+    }
     system_check(gcov_path,
-      args = c(gcov_args, src, "-p", "-o", dirname(src)),
```

Comment:
This should be:
`c(gcov_args, src, "-p", if (gcov_br) "-b", "-o", dirname(src))`

Reply:
Yes, that's much simpler than 

```
if(gcov_br) {
      args = c(gcov_args, src, "-p", "-b", "-c", "-o", dirname(src))
    } else {
      args = c(gcov_args, src, "-p", "-o", dirname(src))
    }
```

I want -c to get the count of execution instead of percentage, so I will change it to

`c(gcov_args, src, "-p", if (gcov_br) "-b", if(gcov_br) "-c", "-o", dirname(src))`